Quadriga/src/main/webapp/WEB-INF/views/addprojectblogentry.jsp

```diff
@@ -35,33 +13,193 @@
+	type="text/javascript"></script>
+<script type="text/javascript">
+	function loadNetwork() {
+		$.ajax({
+			type : "GET",
+			contentType : "application/json",
+			datatype : 'text',
+			url : "${pageContext.servletContext.contextPath}/sites/"
+					+ "sampleproj" + "/visualizenetwork/" + 'NETCEnz9o',
+			timeout : 100000,
+			success : function(data) {
+				visualizeNetwork(data);
+			},
+			error : function(e) {
+				console.log(e.Message);
+			}
+		});
+
+	}
+
+	function visualizeNetwork(jsonString) {
+
+		var container = document.getElementById('networkBox');
+
+		var cy = cytoscape({
+			container : container, // container to render in
+
+			layout : {
+				name : 'cose',
+				idealEdgeLength : 5
+			},
+
+			elements : eval(jsonString),
+
+			style : [
+					{
+						selector : 'node',
+						style : {
+							'background-color' : 'mapData(group, 0, 1, #E1CE7A, #FDD692)',
+							'border-color' : '#B98F88',
+							'border-width' : 1,
+							'font-family' : 'Open Sans',
+							'font-size' : '12px',
+							'font-weight' : 'bold',
+							'color' : 'black',
+							'label' : 'data(conceptName)',
+							'width' : 'mapData(group, 0, 1, 40, 55)',
+							"height" : "mapData(group, 0, 1, 40, 55)",
+							'text-valign' : 'center',
+						}
+					},
+
+					{
+						selector : 'edge',
+						style : {
+							'width' : 1,
+							'line-color' : '#754F44',
+							'target-arrow-shape' : 'none'
+						}
+					} ]
+		});
+
+		defineListeners(cy, '${pageContext.servletContext.contextPath}',
+				'${unixName}');
+
+		$(document)
+				.ready(
+						function() {
+							$('#addImage')
+									.on(
+											'click',
+											function() {
+												var png = cy.png({
+													'scale' : 0.75
+												});
+												tinyMCE
+														.execCommand(
+																'mceInsertContent',
+																false,
+																'<img alt="Smiley face" src="' + png + '"/>');
+											});
+						});
+	}
+</script>
+
+<script src="//cdn.tinymce.com/4/tinymce.min.js" type="text/javascript"></script>
+<!-- Script to generate rich text editor -->
+<script type="text/javascript">
+	tinymce
+			.init({
+				selector : '#description',
+				height : 300,
+				fontsize_formats : "8pt 9pt 10pt 11pt 12pt 14pt 16pt 18pt 20pt 22pt 24pt 48pt 72pt",
+				theme : 'modern',
+				plugins : 'advlist autolink save link image imagetools lists charmap print preview',
+				menubar : false,
+				toolbar : 'undo redo | fontsizeselect | fontselect | styleselect | bold italic | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image | print preview media fullpage | forecolor backcolor emoticons',
+				setup : function(ed) {
+					ed.on('init', function() {
+						this.getDoc().body.style.fontSize = '14px';
+					});
+				}
+
+			});
+</script>
+<script src="/quadriga/resources/js/d3.min.js" charset="utf-8"
+	type="text/javascript"></script>
 <!-- /Content -->
```

Comment:
I am using the default formatter for jsps after replacing tabs with spaces. Is there anything that I need to change in the formatter?